src/pages/Interferometry/MachZehnderEquipment.js

```diff
@@ -117,21 +117,26 @@ class MachZehnderEquipment {
 
   fire(context, photon, photonFire, probability) {
     if (photonFire) {
+      // photon movement
```

Comment:
we don't need this comment because it's clear that the photon moves

Reply:
removed!